openforcefield/utils/toolkits.py

```diff
@@ -1331,7 +1331,7 @@ def to_openeye(molecule, aromaticity_model=DEFAULT_AROMATICITY_MODEL):
         for atom in molecule.atoms:
             oeatom = oemol.NewAtom(atom.atomic_number)
             oeatom.SetFormalCharge(atom.formal_charge.value_in_unit(unit.elementary_charge)) # simtk.unit.Quantity(1, unit.elementary_charge)
-            oeatom.SetAromatic(atom.is_aromatic)
+            #oeatom.SetAromatic(atom.is_aromatic)
```

Comment:
Please add a note here if we intend to un-comment it in the future, or (better, I think) remove this altogether since it does not fit into the medium- or long-term plans you proposed

Reply:
Added a TODO item to revisit the relationship between OFFMols' and OEMols' aromatic flags in a96274f